modules/network-observability-netobserv-cli-reference.adoc

```diff
@@ -1,10 +1,11 @@
-// Module included in the following assemblies:
-// * observability/network_observability/netobserv-cli-reference.adoc
-
+// Automatically generated by './scripts/generate-doc.sh'. Do not edit, or make the NETOBSERV team aware of the editions.
 :_mod-docs-content-type: REFERENCE
-[id="network-observability-netobserv-cli-reference_{context}"]
-= oc netobserv CLI reference 
-The Network Observability CLI (`oc netobserv`) is a CLI tool for capturing flow data and packet data for further analysis.
+[id="network-observability-cli-usage_{context}"]
+= Network Observability CLI usage
+
+You can use the The Network Observability CLI (`oc netobserv`) to pass command line arguments to capture flow data and packet data for further analysis, enable Network Observability Operator features or pass configuration options to the eBPF agent and flowlogs-pipeline.
+== Syntax 
```

Comment:
Need to add extra line before here I think. `== Syntax` is in the same line as L6